### PR TITLE
Spanish translation/Traducción al español

### PR DESCRIPTION
**Pull request by aslozada**
``po`` files translated 
- [x]  `` index.po ``
- [x] `` tutorial.po ``
- [x] `` external.po ``
- [ ] `` how-to.po ``

### CODE REVIEW

File: doc/locales/es/LC_MESSAGES/index.po [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+msgstr "Enlaces externos a otros recursos, blogs, etc."
+
+#: ../doc/index.rst:106
+msgid ""
+"TOML Fortran is used for example in the Fortran package manager (`fpm "
+"<https://fpm.fortran-lang.org>`_), a typical package manifest specified "
+"in TOML is shown below"
+msgstr ""
+"TOML Fortran es usado por ejemplo en el Administrador de paquetes de Fortran (`fpm "
+"<https://fpm.fortran-lang.org>`_). Un manifiesto típico para un paquete especificado en "
```
Comment (awvwgk) [inline doc/locales/es/LC_MESSAGES/index.po:131]: You can directly link to the Spanish translation

```suggestion
"<https://fpm.fortran-lang.org/es>`_). Un manifiesto típico para un paquete especificado en "
```

**Comment by aslozada** [comment]
> @aslozada Let me know when this is ready to get merged. Preferably we can get a review from a Spanish speaker, maybe you know somebody in the Fortran community who is interested to help out here?

Hi Sebastian,
yes, it can be merged. I'm finishing the translation of ```how-to.po```, it's quite extensive.
Github returns a restriction, called DCO (sign off). I didn't understand exactly what it is.

**Comment by awvwgk** [comment]
>  Github returns a restriction, called DCO (sign off). I didn't understand exactly what it is.

Checkout the contributing guidelines on the [developer certificate of origin](https://github.com/toml-f/toml-f/blob/main/CONTRIBUTING.md#sign-your-work). I'm requiring this for all my personal and scientific open source projects to ensure that I'm allowed to distribute contributions under the project's open source license. You of course keep the copyright to your contribution.

**Comment by aslozada** [comment]
Sebastian, just to confirm.  The merge will always be given by you?

--
on the page (the main) https://toml-f.readthedocs.io/en/latest/ the language buttons are missing?

**Comment by awvwgk** [comment]
> Sebastian, just to confirm. The merge will always be given by you?

At the moment it's only me with access to this repo, factually I'll be me merging any patch. However, I'm hoping to find co-maintainers to keep TOML Fortran running in the long term, which is the reason for having it in a separate organization rather my personal namespace in the first place.

I opened an issue for discussing the future of TOML Fortran at https://github.com/toml-f/toml-f/issues/62.

> on the page (the main) https://toml-f.readthedocs.io/en/latest/ the language buttons are missing?

The language navigation is a bit hidden, checkout the *ReadTheDocs* tab in the left sidebar, the first row allow to select the language. Something more prominent like in the fpm docs would be nicer, but I haven't yet checked how to do this on readthedocs.

**Comment by awvwgk** [comment]
Sorry for the delay, I was hoping for further feedback from the discourse thread and forgot to pick this up again.

**Pull request merged**